services/mongodb.service.ts

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable */
+import { Db, Filter, MongoClient, ObjectId, Sort, WithId } from 'mongodb'
+import { Document } from 'bson'
+import { DatabaseCollections } from '../enums/database-collection.enum'
+import { MongoDocument } from '../interfaces/mongo-document.interface'
+
+export class MongodbService {
+    private database: Db
+
+    private async connectToDatabase(): Promise<void> {
+        if (this.database)
+            return
+
+        // eslint-disable-next-line max-len
+        const uri = 'mongodb+srv://admin:admin@portfoliowebsite-db.j0puk.mongodb.net/myFirstDatabase?retryWrites=true&w=majority'
```

Comment:
Sensitive information in source code. Google Environment Variables in node (process.env, dotenv, WebStorm Run Configurations)